zcash_client_backend/src/wallet.rs

```diff
@@ -40,6 +47,20 @@ pub struct WalletTx<N> {
     pub shielded_outputs: Vec<WalletShieldedOutput<N>>,
 }
 
+#[cfg(feature = "transparent-inputs")]
+pub struct WalletTransparentOutput {
+    pub outpoint: OutPoint,
+    pub txout: TxOut,
+    pub height: BlockHeight,
```

Comment:
Is this robust against chain rollbacks?

Reply:
It's used to identify when we need to discard data due to a rollback. So I'm not sure "robust against" is the right way to put it, but yes, rollbacks are taken into account.